### ISSUE TITLE
More explicit error messages

### ISSUE THREAD

**Issue by brpaz** [eclass/semantic-release-docker#460]
Hi. First of all, thank you for this great project.

One of my pain points when using it, are the error messages, which are very generic. For example:

```
 [3:47:28 PM] [semantic-release] › ✔  Completed step "fail" of plugin "@semantic-release/github"
[3:47:28 PM] [semantic-release] › ✖  EDOCKERIMAGETAG Error tag docker image.
Error tag docker image. Check credentials in registries.
```

With this, I only know there was an error tagging the image, but I have no idea what the error was.

I was like 2h of trial and error, in a project before understanding that I was defining the "baseImageTag" property in the wrong format.

I think it would be more helpful to output the error message directly from the docker daemon.

I say, that, the original error is being supressed in the code:

```js
 } catch (err) {
    throw new AggregateError([getError('EDOCKERIMAGETAG', ctx)])
  }
```

**Comment by jgoldschrafe** [comment]
I agree with the above; I imagine this was done this way to prevent Docker repository credentials from leaking into error messages, but this is suppressing too much useful context by swallowing the entire error.

**Comment by meesvandongen** [comment]
For anyone coming across a similar error; I got this error because the image I built in my pipeline didn't have the same name as the specified `baseImageName`.

**Comment by yevon** [comment]
Same here, having errors while tagging images without knowing why it fails.

**Comment by meesvandongen** [comment]
@yevon You could temporarily try my branch `npm i https://github.com/meesvandongen/semantic-release-docker.git` to get a - probably - slightly more useful error.

**Comment by yevon** [comment]
> @yevon You could temporarily try my branch `npm i https://github.com/meesvandongen/semantic-release-docker.git` to get a - probably - slightly more useful error.

How do I reference to your repository in the .releasec.yml?

 I don't really understand what I have to put as baseImagename and imageName, i've tried everything.

**Comment by meesvandongen** [comment]
@yevon `@eclass/semantic-release-docker` should be fine.

Note however that @lgaticaq just merged in the change in said repository, so it should be available via a release. (though there seems to be some issue with prettier eslint rule in combination with the generated changelog - preventing a release for now)

**Comment by yevon** [comment]
Just corrected the issue thanks!, stupid error with gitlab pipelines, it was unable to tag the image because I was executing docker build and semantic-release in different steps.

**Comment by lgaticaq** [comment]
Fixed in [v3.0.1](https://github.com/eclass/semantic-release-docker/releases/tag/v3.0.1). Thanks @meesvandongen
